Exceptions printed when used

```
run-mmm-kafdrop-1    | 2022-01-08 02:01:35.387 ERROR 1 [  XNIO-1 task-5] i.u.s.a.LoggingExceptionHandler          : UT005023: Exception handling request to /error
run-mmm-kafdrop-1    |
run-mmm-kafdrop-1    | java.lang.RuntimeException: org.springframework.web.util.NestedServletException: Request processing failed; nested exception is freemarker.core.InvalidReferenceException: The following has evaluated to null or missing:
run-mmm-kafdrop-1    | ==> error.message  [in template "error.ftlh" at line 6, column 65]
run-mmm-kafdrop-1    |
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | Tip: It's the step after the last dot that caused this error, not those before it.
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | Tip: If the failing expression is known to legally refer to something that's sometimes null or missing, either specify a default value like myOptionalVar!myDefault, or use <#if myOptionalVar??>when-present<#else>when-missing</#if>. (These only cover the last step of the expression; to cover the whole expression, use parenthesis: (myOptionalVar.foo)!myDefault, (myOptionalVar.foo)??
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    |
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | FTL stack trace ("~" means nesting-related):
run-mmm-kafdrop-1    | 	- Failed at: ${error.message}  [in template "error.ftlh" at line 6, column 63]
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | 	at io.undertow.servlet.spec.HttpServletResponseImpl.doErrorDispatch(HttpServletResponseImpl.java:167)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:282)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:79)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:134)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:131)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:255)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:79)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:100)
run-mmm-kafdrop-1    | 	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:387)
run-mmm-kafdrop-1    | 	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:852)
run-mmm-kafdrop-1    | 	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
run-mmm-kafdrop-1    | 	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2019)
run-mmm-kafdrop-1    | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1558)
run-mmm-kafdrop-1    | 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1449)
run-mmm-kafdrop-1    | 	at org.xnio.XnioWorker$WorkerThreadFactory$1$1.run(XnioWorker.java:1280)
run-mmm-kafdrop-1    | 	at java.base/java.lang.Thread.run(Thread.java:829)
run-mmm-kafdrop-1    | Caused by: org.springframework.web.util.NestedServletException: Request processing failed; nested exception is freemarker.core.InvalidReferenceException: The following has evaluated to null or missing:
run-mmm-kafdrop-1    | ==> error.message  [in template "error.ftlh" at line 6, column 65]
run-mmm-kafdrop-1    |
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | Tip: It's the step after the last dot that caused this error, not those before it.
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | Tip: If the failing expression is known to legally refer to something that's sometimes null or missing, either specify a default value like myOptionalVar!myDefault, or use <#if myOptionalVar??>when-present<#else>when-missing</#if>. (These only cover the last step of the expression; to cover the whole expression, use parenthesis: (myOptionalVar.foo)!myDefault, (myOptionalVar.foo)??
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    |
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | FTL stack trace ("~" means nesting-related):
run-mmm-kafdrop-1    | 	- Failed at: ${error.message}  [in template "error.ftlh" at line 6, column 63]
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898)
run-mmm-kafdrop-1    | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:497)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
run-mmm-kafdrop-1    | 	at javax.servlet.http.HttpServlet.service(HttpServlet.java:584)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletHandler.handleRequest(ServletHandler.java:74)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:129)
run-mmm-kafdrop-1    | 	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100)
run-mmm-kafdrop-1    | 	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
run-mmm-kafdrop-1    | 	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
run-mmm-kafdrop-1    | 	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:102)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.RedirectDirHandler.handleRequest(RedirectDirHandler.java:68)
run-mmm-kafdrop-1    | 	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
run-mmm-kafdrop-1    | 	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:257)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchToPath(ServletInitialHandler.java:182)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.spec.RequestDispatcherImpl.error(RequestDispatcherImpl.java:504)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.spec.RequestDispatcherImpl.error(RequestDispatcherImpl.java:417)
run-mmm-kafdrop-1    | 	at io.undertow.servlet.spec.HttpServletResponseImpl.doErrorDispatch(HttpServletResponseImpl.java:165)
run-mmm-kafdrop-1    | 	... 17 more
run-mmm-kafdrop-1    | Caused by: freemarker.core.InvalidReferenceException: The following has evaluated to null or missing:
run-mmm-kafdrop-1    | ==> error.message  [in template "error.ftlh" at line 6, column 65]
run-mmm-kafdrop-1    |
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | Tip: It's the step after the last dot that caused this error, not those before it.
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | Tip: If the failing expression is known to legally refer to something that's sometimes null or missing, either specify a default value like myOptionalVar!myDefault, or use <#if myOptionalVar??>when-present<#else>when-missing</#if>. (These only cover the last step of the expression; to cover the whole expression, use parenthesis: (myOptionalVar.foo)!myDefault, (myOptionalVar.foo)??
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    |
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | FTL stack trace ("~" means nesting-related):
run-mmm-kafdrop-1    | 	- Failed at: ${error.message}  [in template "error.ftlh" at line 6, column 63]
run-mmm-kafdrop-1    | ----
run-mmm-kafdrop-1    | 	at freemarker.core.InvalidReferenceException.getInstance(InvalidReferenceException.java:134)
run-mmm-kafdrop-1    | 	at freemarker.core.EvalUtil.coerceModelToTextualCommon(EvalUtil.java:481)
run-mmm-kafdrop-1    | 	at freemarker.core.EvalUtil.coerceModelToStringOrMarkup(EvalUtil.java:401)
run-mmm-kafdrop-1    | 	at freemarker.core.EvalUtil.coerceModelToStringOrMarkup(EvalUtil.java:370)
run-mmm-kafdrop-1    | 	at freemarker.core.DollarVariable.calculateInterpolatedStringOrMarkup(DollarVariable.java:100)
run-mmm-kafdrop-1    | 	at freemarker.core.DollarVariable.accept(DollarVariable.java:63)
run-mmm-kafdrop-1    | 	at freemarker.core.Environment.visit(Environment.java:347)
run-mmm-kafdrop-1    | 	at freemarker.core.Environment.visit(Environment.java:353)
run-mmm-kafdrop-1    | 	at freemarker.core.Environment.process(Environment.java:326)
run-mmm-kafdrop-1    | 	at freemarker.template.Template.process(Template.java:383)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.view.freemarker.FreeMarkerView.processTemplate(FreeMarkerView.java:391)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.view.freemarker.FreeMarkerView.doRender(FreeMarkerView.java:304)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.view.freemarker.FreeMarkerView.renderMergedTemplateModel(FreeMarkerView.java:255)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.view.AbstractTemplateView.renderMergedOutputModel(AbstractTemplateView.java:179)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.view.AbstractView.render(AbstractView.java:316)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.DispatcherServlet.render(DispatcherServlet.java:1401)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.DispatcherServlet.processDispatchResult(DispatcherServlet.java:1145)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1084)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963)
run-mmm-kafdrop-1    | 	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006)
run-mmm-kafdrop-1    | 	... 45 more
run-mmm-kafdrop-1    |

```

--- a/src/main/java/kafdrop/controller/BasicErrorController.java
+++ b/src/main/java/kafdrop/controller/BasicErrorController.java
@@ -22,8 +22,15 @@ public final class BasicErrorController extends AbstractErrorController {
 
   @RequestMapping("/error")
   public ModelAndView handleError(HttpServletRequest request) {
-    final var error = getErrorAttributes(request, ErrorAttributeOptions.of(ErrorAttributeOptions.Include.STACK_TRACE));
+    final var errorAttributeOptions = ErrorAttributeOptions.of(
+      ErrorAttributeOptions.Include.STACK_TRACE,
+      ErrorAttributeOptions.Include.MESSAGE);
+
+    final var error = getErrorAttributes(request, errorAttributeOptions);
     LOG.info("errorAtts: {}", error);
+    
+    error.putIfAbsent("message", "");
+
     final var model = Map.of("error", error);
     return new ModelAndView("error", model);
   }

